### PR TITLE
Many updates on our large list

### PR DESCRIPTION
**Pull request by TheHollidayInn**

### CODE REVIEW

File: website/client/components/achievements/welcome.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<template lang="pug">
+  b-modal#welcome(:title="$t('welcomeToHabit')", size='lg', :hide-footer="true")
+    .modal-body.container-fluid
+      .row
+        .col-4.col-centered
+          span(style='display:flex')
```
Comment (paglias) [inline website/client/components/achievements/welcome.vue:6]: can we try not to use inline styles?

File: website/client/components/appFooter.vue [hunk 2]
```diff
@@ -1,46 +1,72 @@
-<template lang="pug">
+<template lang='pug'>
```
Comment (paglias) [inline website/client/components/appFooter.vue:1]: I don't remember exactly why but initally we had problems with vue + pug when using single quotes for attributes

File: website/client/components/appFooter.vue [hunk 3]
```diff
@@ -99,125 +142,103 @@
...
 
 <script>
+import axios from 'axios';
+import moment from 'moment';
+import { mapState } from 'client/libs/store';
 import gryphon from 'assets/svg/gryphon.svg';
 
+import modifyInventory from './modifyInventory';
+
 // const IS_PRODUCTION = process.env.NODE_ENV === 'production'; // eslint-disable-line no-process-env
```
Comment (paglias) [inline website/client/components/appFooter.vue:152]: we should import this with `import()` when calling modifyInventory() so it's not loaded for all users

File: website/client/components/appFooter.vue (hunk 2 above)
Comment (TheHollidayInn) [reply website/client/components/appFooter.vue:1]: Yea, when there are single quotes, the syntax highlighter doesn't really work

File: website/client/components/achievements/welcome.vue (hunk 1 above)
Comment (TheHollidayInn) [reply website/client/components/achievements/welcome.vue:6]: Just copied from the live site. I must have missed this.

**Pull request merged**